Android/app/src/main/java/com/example/samplestickerapp/StickerContentProvider.java

```diff
@@ -48,6 +48,8 @@
     public static final String LICENSE_AGREENMENT_WEBSITE = "sticker_pack_license_agreement_website";
     public static final String IMAGE_DATA_VERSION = "image_data_version";
     public static final String AVOID_CACHE = "whatsapp_will_not_cache_stickers";
+    public static final String ANIMATED_STICKER = "animated_sticker_pack";
```

Comment:
I don't see the usage of this, maybe it should be somewhere instead of STICKER_FILE_ANIMATED? for example in `getStickerPackInfo()` ?